gcc/config/riscv/riscv.md

```diff
@@ -390,6 +390,7 @@
 ;; vsmul       vector single-width fractional multiply with rounding and saturation instructions
 ;; vsshift     vector single-width scaling shift instructions
 ;; vnclip      vector narrowing fixed-point clip instructions
+;; vfnrclip     vector fp32 to int8 ranged clip instructions
```

Comment:
They are not standard instruction, so plz create a new section in the comment rather than mixed in the std section, also plz prefixed with `sf_`